subset/pentests/brute_client.py

```diff
@@ -0,0 +1,79 @@
+import socket, sys, json
+
+user_data=open('user_list.json')
+user_list = json.load(user_data)
+user_data.close()
+
+#username=['root','admin','manager']
+username= user_list['username'].split(",")
+
+#password=['root','pass','friend']
+password= user_list['password'].split(",")
+
+arguments = sys.argv
+
+print 'brute_client test begin'
+print 'address:' + arguments[1]
+print 'port:' + arguments[2]
+print 'reportFile:' + arguments[3]
+
+#address = '127.0.0.1'
+address = str(arguments[1])
+
+#port = 10000
+port = int(arguments[2])
+
+report_file = str(arguments[3])
+data_length =64
+
+expected=['login:','Password:','Last login:','Login incorrect','Connection closed by foreign host.']
+expected1=['username:','password:','success','login failed','Connection closed by foreign host.']
```

Comment:
Is expected1 used anywhere?   If so, please try a more meaningful name...

Reply:
changed to expected_alt , code will still have further development in future to improve functionality